src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasServer.kt

```diff
@@ -71,17 +72,24 @@ fun main(args: Array<String>) = mainBody {
     val parser = ArgParser(args)
     val arg = CosmasServerArgs(parser)
     LOG.info("Try to bind in port ${arg.port}")
+    val freeBucket = arg.freeBucket
+    val paidBucket = arg.paidBucket
+    val gsutilImageName = arg.gsutilImageName
     val server =
-            if (arg.bucket != "") {
+            if (freeBucket != null && paidBucket != null) {
                 if (arg.certChain != null && arg.privateKey != null) {
-                    CosmasServer(arg.port, CosmasGoogleCloudService(arg.bucket),
-                            File(arg.certChain), File(arg.privateKey))
+                    CosmasServer(arg.port,
+                            CosmasGoogleCloudService(freeBucket, paidBucket, gsutilImageName = gsutilImageName),
+                            File(arg.certChain),
+                            File(arg.privateKey))
                 } else {
-                    CosmasServer(arg.port, CosmasGoogleCloudService(arg.bucket))
+                    CosmasServer(arg.port,
+                            CosmasGoogleCloudService(freeBucket, paidBucket, gsutilImageName = gsutilImageName))
                 }
             } else {
                 CosmasServer(arg.port, CosmasInMemoryService())

```

Comment:
Please log a message saying that we start in-memory Cosmas (and probably advise like "To run Cosmas with GCS file storage please specify --free-bucket and --paid-bucket arguments")

Reply:
done